GOVERNANCE-elections.md

```diff
@@ -0,0 +1,274 @@
+
+Meeting notes are available to members of the
+[developer mailing list](TODO).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance or community procedures are posted as
+[issues in the community repo](TODO), and
+the Steering Committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their
+employer.
+
+The current membership of the committee is currently (listed alphabetically by
+first name):
+
+| &nbsp;                                                         | Member           | Organization |
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+
+## Decision process
+
+The Steering Committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, creating, removing, or modifying a working group, all spending,
+hiring, and contracting, official responses to publicly raised issues, or any
+other decisions that at least two of the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of more than half of the members
+are present (and all members have been informed of the meeting), and may pass
+with more than half the members of the committee supporting it.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the Steering Committee at
+   [steering@project.org](TODO).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on [the community repository](TODO) and indicate that you would like
+   attention from the steering committee.
+
+## Composition
+
+The steering committee has [TODO: Number] seats. These five seats are
+open to any project contributor. See
+[candidate eligibility](#candidate-eligibility) for a definition.
+
+Steering Committee members serve for 2-year terms, staggered in order to
+preserve continuity. Every year either [TODO:Number] or [TODO:Remainder]
+contributor seats are elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions. One week before the election the call for nominations and exceptions
+will be closed. The election will be open for voting not less than two weeks and
+not more than four. The results of the election will be announced within one
+week of closing the election. New Steering Committee members will take office on
+[TODO: Date] of each year.
+
```

Comment:
The transition process is top of mind right now, so is it necessary to define how the transition will take place and the timeline for said transition?

Reply:
Again, see above.  We simply can't provide for all of the variations of a possible transition in a simple template.